<ui version="4.0" >
 <class>FloorPlan</class>
 <widget class="QMainWindow" name="FloorPlan" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Floor Plan</string>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFicheiro" >
    <property name="title" >
     <string>&amp;Projecto</string>
    </property>
    <widget class="QMenu" name="menu_Novo" >
     <property name="title" >
      <string>&amp;Novo</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuRecentes" >
     <property name="title" >
      <string>&amp;Recentes</string>
     </property>
    </widget>
    <addaction name="menu_Novo" />
    <addaction name="actionAbrir" />
    <addaction name="menuRecentes" />
    <addaction name="separator" />
    <addaction name="actionSair" />
   </widget>
   <widget class="QMenu" name="menuOpera_es" >
    <property name="title" >
     <string>&amp;Operações</string>
    </property>
    <addaction name="actionImprimir" />
    <addaction name="actionCopiar" />
    <addaction name="actionGuardar_como_imagem" />
    <addaction name="actionMemoria_Descritiva" />
    <addaction name="separator" />
    <addaction name="actionAumentar_Zoom" />
    <addaction name="actionDiminuir_Zoom" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuAcerca" >
    <property name="title" >
     <string>&amp;Ajuda</string>
    </property>
    <addaction name="actionAjuda" />
    <addaction name="separator" />
    <addaction name="actionDocumenta_o" />
    <addaction name="separator" />
    <addaction name="actionAcerca_de_FloorPlan" />
    <addaction name="actionAcerca_do_Qt" />
   </widget>
   <widget class="QMenu" name="menuPre_ferencias" >
    <property name="title" >
     <string>Pre&amp;ferencias</string>
    </property>
   </widget>
   <addaction name="menuFicheiro" />
   <addaction name="menuOpera_es" />
   <addaction name="menuPre_ferencias" />
   <addaction name="menuAcerca" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QDockWidget" name="SvgList" >
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize" >
    <size>
     <width>200</width>
     <height>600</height>
    </size>
   </property>
   <property name="features" >
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable</set>
   </property>
   <property name="windowTitle" >
    <string>Lista</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <widget class="QScrollArea" name="scrollArea" >
     <property name="geometry" >
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>1000</height>
      </rect>
     </property>
     <property name="minimumSize" >
      <size>
       <width>150</width>
       <height>1000</height>
      </size>
     </property>
     <property name="widgetResizable" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>196</width>
        <height>996</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionAbrir" >
   <property name="text" >
    <string>&amp;Abrir</string>
   </property>
  </action>
  <action name="actionSair" >
   <property name="text" >
    <string>&amp;Sair</string>
   </property>
  </action>
  <action name="actionImprimir" >
   <property name="text" >
    <string>Imprimir</string>
   </property>
  </action>
  <action name="actionGuardar_como_imagem" >
   <property name="text" >
    <string>Guardar como imagem</string>
   </property>
  </action>
  <action name="actionCopiar" >
   <property name="text" >
    <string>Copiar</string>
   </property>
  </action>
  <action name="actionDocumenta_o" >
   <property name="text" >
    <string>Documentação</string>
   </property>
  </action>
  <action name="actionAcerca_de_FloorPlan" >
   <property name="text" >
    <string>Acerca de FloorPlan</string>
   </property>
  </action>
  <action name="actionAcerca_do_Qt" >
   <property name="text" >
    <string>Acerca do Qt</string>
   </property>
  </action>
  <action name="actionAjuda" >
   <property name="text" >
    <string>Ajuda</string>
   </property>
  </action>
  <action name="actionMemoria_Descritiva" >
   <property name="text" >
    <string>Memoria Descritiva</string>
   </property>
  </action>
  <action name="actionAumentar_Zoom" >
   <property name="text" >
    <string>Aumentar Zoom </string>
   </property>
  </action>
  <action name="actionDiminuir_Zoom" >
   <property name="text" >
    <string>Diminuir Zoom</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
